acceptance/tests/reports/report_storage.rb

```diff
@@ -2,14 +2,14 @@
 
 test_name "basic validation of puppet report submission" do
 
-  step "setup a test manifest for the master and perform agent runs" do
-    manifest = <<-MANIFEST
+  manifest = <<-MANIFEST
```

Comment:
I'm not sure we should be overloading noop related testing with an existing test here. I think a new test is in order.